templates/new/package.json

```diff
@@ -37,7 +37,7 @@
     "redux": "3.6.0",
     "sass-loader": "3.2.0",
     "style-loader": "0.13.1",
-    "webpack": "1.13.2"
+    "webpack": "2.2.0"
```

Comment:
This looks right to me, is there a reason package.json at the top doesn't use this version too?